modules/planning/tasks/rss/decider_rss.cc

```diff
@@ -228,22 +245,23 @@ Status RssDecider::Process(Frame *frame,
   if (front_obstacle_distance <
       kTrajectoryLenFactor * reference_line_info->TrajectoryLength()) {
     reference_line_info->mutable_rss_info()->set_cur_dist_lon(
-        front_obstacle_distance);
+      front_obstacle_distance);
   }
 
-  reference_line_info->mutable_rss_info()->set_rss_safe_dist_lon(dMin_lon);
+  reference_line_info->mutable_rss_info()->set_rss_safe_dist_lon(
+      double(dMin_lon));
   reference_line_info->mutable_rss_info()->set_acc_lon_range_minimum(
-      accelerationRestriction.longitudinalRange.minimum);
+      double(accelerationRestriction.longitudinalRange.minimum));
   reference_line_info->mutable_rss_info()->set_acc_lon_range_maximum(
-      accelerationRestriction.longitudinalRange.maximum);
+      double(accelerationRestriction.longitudinalRange.maximum));
   reference_line_info->mutable_rss_info()->set_acc_lat_left_range_minimum(
-      accelerationRestriction.lateralLeftRange.minimum);
+      double(accelerationRestriction.lateralLeftRange.minimum));
   reference_line_info->mutable_rss_info()->set_acc_lat_left_range_maximum(
-      accelerationRestriction.lateralLeftRange.maximum);
+      double(accelerationRestriction.lateralLeftRange.maximum));
   reference_line_info->mutable_rss_info()->set_acc_lat_right_range_minimum(
-      accelerationRestriction.lateralRightRange.minimum);
+      double(accelerationRestriction.lateralRightRange.minimum));
   reference_line_info->mutable_rss_info()->set_acc_lat_right_range_maximum(
-      accelerationRestriction.lateralRightRange.maximum);
+      double(accelerationRestriction.lateralRightRange.maximum));
```

Comment:
1. This file failed to pass lint check. You may try `apollo.sh lint` to find these errors, or use tool `scripts/clang-format.sh` to fix it automatically.
2. Use `static_cast<double>` to cast scala types.

Reply:
@xiaoxq ,  thanks for your comment.  this issue is resolved in the patch "rss-open-source-library-fix-cpplint-issue"

Reply:
@xiaoxq could you change the review status and the current status is still "request changes".